docs/user_docs/cli/kbcli_cluster_create.md

```diff
@@ -89,34 +89,34 @@ kbcli cluster create [NAME] [flags]
-      --enable-all-logs                        Enable advanced application all log extraction, set to true will ignore enabledLogs of component level, default is false
-  -h, --help                                   help for create
-      --monitoring-interval uint8              The monitoring interval of cluster, 0 is disabled, the unit is second, any non-zero value means enabling monitoring.
-      --node-labels stringToString             Node label selector (default [])
-  -o, --output format                          Prints the output in the specified format. Allowed values: JSON and YAML (default yaml)
-      --pitr-enabled                           Specify whether enabled point in time recovery
-      --pod-anti-affinity string               Pod anti-affinity type, one of: (Preferred, Required) (default "Preferred")
-      --pvc stringArray                        Set the cluster detail persistent volume claim, each '--pvc' corresponds to a component, and will override the simple configurations about storage by --set (e.g. --pvc type=mysql,name=data,mode=ReadWriteOnce,size=20Gi --pvc type=mysql,name=log,mode=ReadWriteOnce,size=1Gi)
-      --rbac-enabled                           Specify whether rbac resources will be created by kbcli, otherwise KubeBlocks server will try to create rbac resources
-      --restore-to-time string                 Set a time for point in time recovery
-      --set stringArray                        Set the cluster resource including cpu, memory, replicas and storage, each set corresponds to a component.(e.g. --set cpu=1,memory=1Gi,replicas=3,storage=20Gi or --set class=general-1c1g)
-  -f, --set-file string                        Use yaml file, URL, or stdin to set the cluster resource
-      --tenancy string                         Tenancy options, one of: (SharedNode, DedicatedNode) (default "SharedNode")
-      --termination-policy string              Termination policy, one of: (DoNotTerminate, Halt, Delete, WipeOut) (default "Delete")
-      --tolerations strings                    Tolerations for cluster, such as "key=value:effect, key:effect", for example '"engineType=mongo:NoSchedule", "diskType:NoSchedule"'
-      --topology-keys stringArray              Topology keys for affinity
-      --volume-restore-policy string           the volume claim restore policy, supported values: [Serial, Parallel] (default "Parallel")
+      --backup string                                          Set a source backup to restore data
+      --backup-cron-expression string                          the cron expression for schedule, the timezone is in UTC. see https://en.wikipedia.org/wiki/Cron.
+      --backup-enabled                                         Specify whether enabled automated backup
+      --backup-method kbcli cd describe <cluster-definition>   the backup method, view it by kbcli cd describe <cluster-definition>, default is volume-snapshot (default "volume-snapshot")
```

Comment:
`--backup-method kbcli cd describe <cluster-definition>` this format is not right.